scene/animation/tween.cpp

```diff
@@ -675,7 +675,7 @@ bool CallbackTweener::step(double &r_delta) {
 		return false;
 	}
 
-	if (!callback.get_object()) {
+	if (!callback.is_valid()) {
```

Comment:
This breaks static methods. For whatever reason their `is_valid()` returns false, but their Object is the source GDScript. Example:
```GDScript
@tool
extends EditorScript

class CustomNode extends Node:
	static func test():
		print(1)

func _run():
	var tw = Engine.get_main_loop().create_tween()
	tw.tween_callback(CustomNode.test)
```
Does not work with this PR.

Reply:
Oh, I meant C++ static callables (`callable_mp_static()`). Regarding GDScript, there's #79521, that I think has to do.

Reply:
Well that issue has to be resolved before these changes.

Reply:
Hrm. This sounds like there's probably a problem with `is_valid()` for static GDScript methods.

The aim of this PR is to fix situations where `callable_mp_static()` is not accepted, which has the exact opposite problem: `get_object()` returns null, but `is_valid()` is true. I think we need to try and standardize on `is_valid()` being the canonical way to check if a callable is valid.

Reply:
I've created #82755 to address the problem with GDScript static methods.